core/templates/domain/exploration/ExplorationObjectFactorySpec.ts

```diff
@@ -102,6 +108,10 @@ describe('Exploration object factory', () => {
           default_outcome: {}
         }
       },
+      classifier_model_id: null,
+      card_is_checkpoint: false,
+      linked_skill_id: null,
+      next_content_id_index: Number(undefined)
```

Comment:
What does this mean?

Reply:
These values are added because the strict typescript checker throws error for missing keys if they are not optional.

Reply:
But why you use `Number(undefined)`

Reply:
I wasn't sure what should be the default value for `next_content_id_index`. Hence, I used `undefined`.

Reply:
Looking at the codebase I think you can use just `1`

Reply:
Done.